src/components/Fields/TextArea.story.js

```diff
@@ -34,13 +34,13 @@ stories.add(
 );
 
 stories.add(
-    'Label Always',
+    'Label On Hover',
```

Comment:
This name doesn't properly describe the prop since it's on hover and on focus.
Perhaps it should be something like "Auto Hiding Label"?